### DISCUSSION TITLE
create a pdf using bun express.js

### DISCUSSION THREAD

**Discussion by 0xtz** [diegomura/react-pdf#2459] [Q&A]
## generating a pdf in express.js bun

hello, i wanna create a pdf generator im using, __bun, Express.js @react-pdf/renderer__ 
after making the set up etc, i got this issue :

![image](https://github.com/diegomura/react-pdf/assets/53442065/f546ed91-6063-49c7-9991-c5bbc6b3ad11)

do i need something in the config ? hmmmm
i did all the setup needed
this is my route :
```
import ReactPDF from "@react-pdf/renderer";
import { NextFunction, Request, Response, Router } from "express";

//
const router: Router = Router();

// 
router.get(
  "/generate-pdf",
  async (req: Request, res: Response, next: NextFunction): Promise<void> => {
    try {
      ReactPDF.render(<MyDocument />, `example.pdf`);

      res.status(200).json({
        message: "Server is up and running",
        success: true,
      });
    } catch (error: unknown) {
      next(new Error((error as Error).message));
    }
  }
);

...

**Comment by theskillwithin** [comment]
How did you resolve?
